packages/web3-constants/evm/explorer.json

```diff
@@ -35,6 +38,9 @@
         "Arbitrum": "https://api.arbiscan.io/api",
         "Arbitrum_Rinkeby": "",
         "xDai": "https://blockscout.com/xdai/mainnet/api",
+        "Optimism": "https://api-Optimism.etherscan.io/api",
+        "Optimism_Kovan": "https://kovan-Optimism.etherscan.io",
+        "Optimism_Goerli": "https://goerli-Optimism.etherscan.io",
```

Comment:
```suggestion
        "Optimism": "https://api-optimism.etherscan.io/api",
        "Optimism_Kovan": "https://kovan-optimism.etherscan.io",
        "Optimism_Goerli": "https://goerli-optimism.etherscan.io",
```

Reply:
Does these links work?

Reply:
correct is: "https://goerli-optimistic.etherscan.io/